CGBEmu/source/Player.cpp

```diff
@@ -1,24 +1,28 @@
+#include "Player.h"
 #include <iostream>
-class Player
+
+Player::Player()
+	: width(0)
+	, height(0)
 {
-	int width;
-	int height;
-public:
-	Player();
-	~Player();
-	void set_values();
-	int area() { return width*height; }
-};
+
+}
 
 Player::~Player()
 {
 	
 }
+
 void Player::set_values() {
 
 	std::cout << "Please enter width: ";
 	std::cin >> width;
 	std::cout << "Please enter height: ";
 	std::cin >> height;
 	std::cout << "area:"<< width*height;
+}
+
+int Player::area() const
```

Comment:
const here means that this function will not change the values of any of the member variables in the class